keyboards/crkbd/keymaps/ninjonas/rules.mk

```
@@ -0,0 +1,18 @@
+RGBLIGHT_ENABLE = yes
+MOUSEKEY_ENABLE = no
+OLED_DRIVER_ENABLE = yes
+
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
+
+EXTRAFLAGS += -flto
```

```suggestion
LINK_TIME_OPTIMIZATION_ENABLE = yes
```

That will do this, and disable some additional stuff that would cause compiler errors normally.   So this may be a simpler solution.